clipper_admin/clipper_admin/clipper_admin.py

```diff
@@ -237,7 +237,8 @@ def build_and_deploy_model(self,
                                labels=None,
                                container_registry=None,
                                num_replicas=1,
-                               batch_size=-1):
+                               batch_size=-1,
+                               pkgs_to_install=None):
```

Comment:
Add this `pkgs_to_install` command arg to the all of the model deployers in `clipper_admin/deployers/`.